src/args.ts

```diff
@@ -0,0 +1,49 @@
+interface iCommands {
```

Comment:
nit: let's remove the `i` prefix, and since this represents one command, let's make it singular.
 
```suggestion
interface Command {
```